app/lib/pre_assembly/file_manifest.rb

```diff
@@ -18,8 +18,13 @@ def initialize(staging_location:, csv_filename:)
       @manifest = load_manifest # this will cache the entire file manifest csv in @manifest
     end
 
-    def exists?
-      File.exist?(csv_filename)
+    def valid?
```

Comment:
the `exists?` method makes no sense here...because we try to load the manifest file itself when we initialize this class, and so if the file doesn't exist, the class would have already thrown an exception on initialize

an attempt to use this method was being made elsewhere in the code when checking to see if a job run was using a file manifest, but it makes more sense to just inspect the `using_file_manifest` attribue on the model, which tells you this directly